wcomponents-theme/unbuilt.package.json

```diff
@@ -18,6 +18,8 @@
 		"handlebars": "~4.0.5",
 		"i18next": "~3.4.1",
 		"i18next-xhr-backend": "~1.0.1",
```

Comment:
Override the versions brought in by intern for "digdug" and "leadfoot" that is using "~". The latest version of these breaks saucelabs.